src/frontends/common/src/manager.cpp

```diff
@@ -245,36 +245,11 @@ FrontEnd::Ptr FrontEndManager::load_by_model(const std::vector<ov::Any>& variant
 namespace ov {
 namespace frontend {
 
-namespace {
-
-class FrontEndManagerHolder {
-    std::mutex m_mutex;
-    std::weak_ptr<FrontEndManager> m_manager;
-
-public:
-    FrontEndManagerHolder(const FrontEndManagerHolder&) = delete;
-    FrontEndManagerHolder& operator=(const FrontEndManagerHolder&) = delete;
-
-    FrontEndManagerHolder() = default;
-
-    FrontEndManager::Ptr get() {
-        std::lock_guard<std::mutex> lock(m_mutex);
-        auto manager = m_manager.lock();
-        if (!manager)
-            m_manager = manager = std::make_shared<FrontEndManager>();
-        return manager;
-    }
-};
-
-FrontEndManagerHolder& get_frontend_manager_holder() {
-    static FrontEndManagerHolder frontend_manager_holder;
-    return frontend_manager_holder;
-}
-
-}  // namespace
-
 FrontEndManager::Ptr get_frontend_manager() {
-    return get_frontend_manager_holder().get();
+    static std::shared_ptr<FrontEndManager> manager = nullptr;
+    if (!manager)
+        manager = std::make_shared<FrontEndManager>();
+    return manager;
```

Comment:
looks like we have not had this function before as well as changes in .hpp file https://github.com/openvinotoolkit/openvino/pull/11650/files#diff-9c901a405d3b2bb03116cbbdf5b52f607139c4c0690fafb81f96355fc02bce9e

Reply:
Original version puts this static Frontend Manager instance in ie_network_reader.cpp, which is a local function. Why we not put it into a global interface to benefit more potential requirement?

Reply:
ok, no problem.

Reply:
done